### ISSUE TITLE
Clarify authorization layer requirements, flows and roles

### ISSUE THREAD

**Issue by awoie** [decentralized-identity/secure-data-store#104]
TL;DR; it is unclear which roles, flows and use cases are envisioned and therefore it is not clear what requirements the authorization layer should address.

Proposal: 
- MUST support one authorization layer with x, y, z requirements where a sub-specification exists (e.g., through registry?)
- authorization layer becomes an "extension point"
- Create separate documents for e.g., ZCaps, OAuth2 (DPoP)

Note, I might withdraw the proposal if it becomes more clear why certain technologies cannot be used.

Disclaimer: I like ZCaps but I also like to use existing implementations. From my experience with extending OAuth2 servers, with most of the them it is not possible to replace the entire token format which will make it quite hard to support ZCaps. If there is a good reason why to use ZCaps, then I'm totally fine with making ZCaps mandatory.

Further, I want to better understand what would be in scope of the SDS spec re ZCaps, e.g., specific caveats etc?

**Comment by agropper** [comment]
Can someone post a link for ZCaps, please?

**Comment by awoie** [comment]
@agropper this is the specification: https://w3c-ccg.github.io/zcap-ld/

**Comment by agropper** [comment]
Thanks, Oliver. I've put a pair of comments
https://github.com/w3c/did-core/issues/382#issuecomment-695346948 and
https://github.com/w3c/did-core/issues/382#issuecomment-695830510

I don't think it's easy to fulfill the Alice to Bob authorization
requirements in this use-case with OAuth2. Can someone familiar with
zcap-ld edit the sequence diagram to show how it could work in this case?
